apps/wappush/js/wappush.js

```diff
@@ -115,8 +119,8 @@
     }).catch(function(error) {
       // If we encountered an error don't process messages
       wapPushEnabled = false;
-      error = error || 'Unknown error';
-      console.error('Could not initialize: ', error);
+      var message = error.message || 'Unknown error';
+      console.error('Could not initialize the WAP push manager: ', message);
       return error;
```

Comment:
looking at this, I wonder if we should not rethrow the error (so that the promise is still rejected) ?